### PR TITLE
Properly shift os.system exit code bits

### PR DESCRIPTION
**Pull request by mckornfield**

### CODE REVIEW

File: aws_export_credentials/aws_export_credentials.py [hunk 1]
```diff
@@ -194,7 +194,11 @@ def main():
             os.environ['AWS_DEFAULT_REGION'] = region_name
 
         command = ' '.join(shlex.quote(arg) for arg in args.exec)
-        sys.exit(os.system(command))
+        code = os.system(command)
```
Comment (mckornfield) [inline aws_export_credentials/aws_export_credentials.py:197]: @benkehoe after doing some testing and driving myself nuts, I finally stumbled on this bit shifting that I needed to do...

**Comment by benkehoe** [comment]
Perhaps it's better to take this opportunity to change it to use `subprocess`?

**Comment by benkehoe** [comment]
Could you give this branch a try? https://github.com/benkehoe/aws-export-credentials/tree/use-subprocess

**Comment by mckornfield** [comment]
Yep that branch works for me 👍 want me to close this PR?

**Comment by benkehoe** [comment]
Yeah, I'll ship this this afternoon

**Comment by benkehoe** [comment]
I shipped this as v0.16 and yanked v0.15 on PyPI. I also removed support for 3.5 and 3.6 as they are end-of-life.

**Pull request closed without merging**